ui/StatusQ/src/StatusQ/Components/StatusSmartIdenticon.qml

```diff
@@ -65,15 +69,20 @@ Loader {
 
     Component {
         id: roundedIcon
-        StatusRoundIcon {
-            asset.bgWidth: root.asset.bgWidth
-            asset.bgHeight: root.asset.bgHeight
-            asset.bgColor: root.asset.bgColor
-            asset.width: root.asset.width
-            asset.height: root.asset.height
-            asset.name: root.asset.name
-            asset.rotation: root.asset.rotation
-            asset.color: root.asset.color
+
+        StatusRoundButton {
+            width: root.asset.bgWidth
+            height: root.asset.bgHeight
+            color: root.asset.bgColor
+            icon.width: root.asset.width
+            icon.height: root.asset.height
+            icon.name: root.asset.name
+            icon.rotation: root.asset.rotation
+            icon.color: root.asset.color
+            icon.hoverColor: root.asset.hoverColor
+            radius: root.asset.bgRadius
+            hoverEnabled: root.hoverEnabled
+            onClicked: root.clicked()
```

Comment:
This could stay `StatusRoundIcon` and instead you could add a `MouseArea` for the whole component, wdyt?

Reply:
See the reasoning above